app/components/global-navigation/search.js

```diff
@@ -430,6 +437,15 @@ export default Component.extend({
 		return this.get('searchScope') === 'cross-wiki' ? 'cross-wiki' : 'internal';
 	},
 
+	getDefaultScope() {
+		const savedSearchScope = Cookies.get('DEFAULT_SEARCH_SCOPE');
```

Comment:
how about just `return Cookies.get('DEFAULT_SEARCH_SCOPE') || 'internal'`

Reply:
I would stick with this little validation. Just because different value will break nav-search